.gitignore

```diff
@@ -30,3 +30,6 @@ data-files/model
 packaged
```

Comment:
The `packaged` directory was my (old) version of what now appears to be `dist`. Since we are moving to a different name we can probably drop this from the `.gitignore` (and I can rename that default directory in my packager script if we prefer `dist`).

Reply:
I really don't know what it should be called... I've gone back and forth on naming, and you can see my script is called "packager". Since the testing infrastructure uses `packages` for the test stuff, I figured having a different name made sense. 

I propose we leave `packages` for external things we're bringing in and `dist` for stuff we're intending to distribute to others.